package.json

```diff
@@ -44,8 +44,8 @@
     "process": "^0.11.10",
     "proxy-polyfill": "^0.3.2",
     "qrcode-generator": "^1.4.4",
-    "react": "17.0.1",
-    "react-native": "0.64.2",
+    "react": "17.0.2",
+    "react-native": "0.65.0",
```

Comment:
Did you try upgrading to the [latest version](https://github.com/facebook/react-native/releases/tag/v0.65.1)?

Reply:
Can you also update `@types/react-native` and `@types/react` to the latest versions?

Reply:
done ✅